### DISCUSSION TITLE
Directory of the downloaded models

### DISCUSSION THREAD

**Discussion by yunxiao-li-psych** [absadiki/subsai#141] [Q&A] [answered]
Hi developers, thank you for developing such a wonderful tool. I want to know the directory of the downloaded models, because I want to remove some of them that I don't need anymore. Thank you for your help!

**Comment by yunxiao-li-psych** [comment, marked as answer]
Ah, I found it. I'm using Mac, and the directory is `~/.cache/whisper`. Hope this will be helpful to others.

**Comment by absadiki** [comment]
Hi @yunxiao-li-psych, 
You are welcome, Glad you found it useful. 

The directory of the downloaded models depends on which Whisper implementation and Operating system you are using.
For the original Whisper, yes you'll find the models in `~/.cache/whisper`.
